CHANGELOG.md

```diff
@@ -49,6 +49,7 @@
         - Allow cobrands to disable admin resending.
         - Sass variables for default link colour and decoration.
         - Make contact edit note optional on staging sites.
+        - Combine fetch-comments and fetch-reports scripts.
```

Comment:
I feel like we should explicitly mention that people with an FMS install will need to update any cronjobs to use the new `bin/fetch` script. Relatedly it would also be good to have a bit in the release notes about the mapping between old and new commands. So `bin/fetch-comments` becomes `bin/fetch --updates`, `bin/fetch-comments-24hs` becomes `bin/fetch --updates --start 24` etc (or I guess could just point them at the example crontab, or the relevant commit?).